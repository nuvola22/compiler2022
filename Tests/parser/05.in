const a = 1;
const b: Integer = 1;
    c = 1 + 1;
    d = 'Hello, world';
begin end.