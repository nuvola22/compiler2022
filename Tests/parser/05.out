      program
         constant
            a
            1
         constant
            b
            Integer
            1
         constant
            c
            +
               1
               1
         constant
            d
            'Hello, world'
         begin
